### PR TITLE
Move `Tools` content into a menu

### PR DESCRIPTION
**Pull request by arielj**
This PR changes the layout of the game page to show the actions that are currently in the Tools tab in a menu triggered by a "hamburger" button on the corner. Hopefully this helps with the discoverability of the options.

Changes:
- Moved most actions into the menu
- Moved store logo to the bottom left
- Moved the Requirements as a menu option and a Dialog
- Moved Wine and WinePrefix path to the main info
- Fixed an issue where the `First/Last/Time Played` was grabbing the controller focus incorrectly.

I tested also with a controller.

image
image

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [ ] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

File: src/frontend/screens/Game/GameSubMenu/index.tsx [hunk 1]
```diff
@@ -383,17 +348,15 @@ export default function GamesSubmenu({
             {t('submenu.protondb')}
           </button>
         )}
+        {onShowRequirements && (
+          <button
+            onClick={async () => onShowRequirements()}
+            className="link button is-text is-link"
+          >
+            {t('game.reuirements', 'Requirements')}
```
Comment (arielj) [inline src/frontend/screens/Game/GameSubMenu/index.tsx:356]: I know there's a typo in `reuirements` but the translation key is already this, I didn't want to change it and break something

**Comment by flavioislima** [comment]
@biliesilva
can you check the design and maybe add a mockup for the position of the Menu?

File: src/frontend/screens/Game/GameSubMenu/index.tsx (hunk 1 above)
Comment (flavioislima) [reply src/frontend/screens/Game/GameSubMenu/index.tsx:356]: not an issue if it is only the key.

**Comment by biliesilva** [comment]
Is it possible to make the hamburger menu a little less big and also give more spacing between the lines of the text? It's easier to read with slightly larger spacing. Also, It would be nice to increase the weight of some parts of the text:
image

**Review by flavioislima — APPROVED** [review]
Looks pretty good codewise and the functionality.
But I agree with Billie that we could try adding the menu in other places to see how it looks.

**Comment by arielj** [comment]
I applied the suggestions, it looks like this now:
image

Any suggestion on where to put the menu button to try alternatives? I tried in other places but they looked really bad to me so I don't know what alternatives you would like to see.

**Comment by arielj** [comment]
Some alternatives:
image
image
image

**Comment by biliesilva** [comment]
The first option is more close to the convention.

**Comment by flavioislima** [comment]
I vote for the first option as well.
I noticed that the Store logo was removed from the game page, maybe we could add it again in another place, maybe with the title or at the bottom of the screen?

**Comment by arielj** [comment]
> I vote for the first option as well. I noticed that the Store logo was removed from the game page, maybe we could add it again in another place, maybe with the title or at the bottom of the screen?

by first option you mean the first I posted (the current code, at the top right corner) or the first alternative (the second comment that is not the current code, at the right side of the game title)?

the store logo is currently at the bottom left, I can put at the bottom right maybe, or put it back at the top right

**Comment by biliesilva** [comment]
> > I vote for the first option as well. I noticed that the Store logo was removed from the game page, maybe we could add it again in another place, maybe with the title or at the bottom of the screen?
> 
> by first option you mean the first I posted (the current code, at the top right corner) or the first alternative (the second comment that is not the current code, at the right side of the game title)?
> 
> the store logo is currently at the bottom left, I can put at the bottom right maybe, or put it back at the top right

I mean this one: 
image

**Pull request merged**